03/Task03.js

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
+import React from "react";
+import GitHub from "../src/modules/github/github/GitHub";
```

Comment:
Wracając do plików `index.js` w katalogu `modules`.
Tam chodzi o to, aby "skrócić" sobie drogę do odpowiedniego elementu.
Pomijając to, że masz niepotrzebnie 2x github to `index.js` pozwoli nam np. w imporcie zapisać `../src/modules/github` zamiast `../src/modules/github/GitHub` - tam po prostu tworzymy importy i eksporty dla elementów, które mają być "bardziej dostępne".